.github/workflows/conformance-ingress.yaml

```diff
@@ -400,7 +400,12 @@ jobs:
         timeout-minutes: 30
         run: |
           cd ingress-controller-conformance
-          ./ingress-controller-conformance -ingress-class cilium -wait-time-for-ingress-status 60s -wait-time-for-ready 60s
+          ./ingress-controller-conformance \
+            -ingress-class cilium \
+            -wait-time-for-ingress-status 60s \
+            -wait-time-for-ready 60s \
+            -enable-http-debug \
+            -stop-on-failure
```

Comment:
Hmm, do we want to stop on failure? I guess it's probably more likely we'll fail everything, but it would be handy to know if we only fail one of the tests...

Reply:
Ideally, yes, we would love to run all the tests by default, and see all failures at once. However, continue running other tests will automatically clean-up the failed test resources, which makes it hard to capture sysdump for further investigation.

Just a note that `test` here is actually feature in ingress conformance, we have only 4 tests/features in total.